src/routes/two-factor-authentication/routes.ts

```diff
@@ -0,0 +1,11 @@
+import * as validatePassword from './validate-password';
```

Comment:
I think the `validatePassword` route should be dropped, for 2 reasons:
* It doesn't add any security. I think it is there to avoid that someone with access to the computer of the user can enable 2FA? In the current setup, such a user could also go to the next step (`twoFactorAuthentication.getKeyuri(..)`) without validating the password first by fiddling with the front-end code.
* there's no user interaction required between the two steps

Instead, I propose to:
* add a `password: string;` property to the getKeyURI route
* have that route throw an error if 2FA is already enabled

Reply:
One afterthought: I think we should also add `password: string` to the `disable` route.

Reply:
Updated